docker/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM alpine:3.8 as release
+
+RUN apk --no-cache add ca-certificates \
+  && apk --no-cache add \
+    curl \
+    tar
+
+RUN mkdir /source
+WORKDIR /source
+RUN curl -Lo strichliste.tar.gz https://github.com/strichliste/server/releases/download/v0.2.3/strichliste.tar.gz
+RUN tar -xf strichliste.tar.gz
+RUN rm -r strichliste.tar.gz
```

Comment:
I would prefer to put the Dockerfile in the project root and use `ADD . /source` and `.dockerignore`. 
Using commands to download files leads to issues with dockers caching. If you really want to download it you can use the `ADD <url>` command.

Reply:
AFAIK `docker build` will use a cache if no change was made to the Dockerfile and the upstream image was not pulled.

Reply:
As we are using versioned releases, which should be immutable, a new release means a change to the dockerfile and therefore clearing the cache. But I might be missing something.